easy-ui-tsconfig/base.json

```diff
@@ -5,7 +5,7 @@
     "isolatedModules": true,
     "experimentalDecorators": true,
     "lib": ["dom", "es2020"],
-    "moduleResolution": "node16",
+    "moduleResolution": "node",
```

Comment:
turns out there are some quirks with `node16` that prevent us from using it smoothly currently. TS 5.0 is supposed to address some of these